src/server.h

```diff
@@ -1799,6 +1799,8 @@ struct valkeyServer {
                                                  invocation of the event loop. */
     unsigned int max_new_conns_per_cycle;     /* The maximum number of tcp connections that will be accepted during each
                                                  invocation of the event loop. */
+    int write_throttling;                     /* 1 if write throttling is enabled: prevents output buffer overflow (and
```

Comment:
```suggestion
    int write_throttling;                     /* 1 if write throttling is enabled: prevents replication buffer overflow (and
```

Reply:
I am interested in this topic, so far it looks like the solution is throwing an error code.